test/api/fixtures.h

```diff
@@ -81,6 +84,9 @@ struct Fixtures {
     // https://www.flickr.com/photos/grizdave/2569067123/ (same as input_jpg)
     std::string input_png_rgb_with_alpha{dir + "/2569067123_aca715a2ee_o.png"};
 
+    // https://images.unsplash.com/photo-1586015910630-34cc709e5f91 (same as input_jpg_saturate)
+    std::string input_png_saturate{dir + "/photo-1586015910630-34cc709e5f91.png"}
+
```

Comment:
Could you not use [`input_png_rgb_with_alpha`](https://raw.githubusercontent.com/weserv/images/5.x/test/api/fixtures/2569067123_aca715a2ee_o.png) instead?